q2_types/per_sample_sequences/tests/test_transformer.py

```diff
@@ -951,5 +956,86 @@ def test_validate_paired_end_fastq_manifest_directions_invalid(self):
             _validate_paired_end_fastq_manifest_directions(manifest)
 
 
+# A quick note about this test suite: we are really only interested in the
+# manifest, since this transformer primarily transforms the new MD-manifest to
```

Comment:
If you leave this comment, include the format name here instead of "new MD-manifest" (since it won't always be new).

Reply:
Pushed up some changes in 2933f4bcce5ebf7a2d5f261882a078501cd93750 --- let me know if that is more clear. I avoided naming all four of those Formats, because they are annoyingly long (and would need to do it for the old set and new, so 8 formats total), so I attempted to rephrase in a way that let's us not write all of those out.